engine/admin/logging/logentries.md

```diff
@@ -0,0 +1,47 @@
+<!--[metadata]>
++++
+aliases = ["/engine/reference/logging/logentries/"]
+title = "Logentries logging driver"
+description = "Describes how to use the logentries logging driver."
+keywords = ["logentries, docker, logging, driver"]
+[menu.main]
+parent = "smn_logging"
++++
+<![end-metadata]-->
```

Comment:
@mstanleyjones I think this metadata needs to be updated for the new publishing system; could you assist here, and show what it should look like?